jmclient/jmclient/wallet_utils.py

```diff
@@ -362,23 +362,30 @@ def get_imported_privkey_branch(wallet_service, m, showprivkey):
         return WalletViewBranch("m/0", m, -1, branchentries=entries)
     return None
 
-def wallet_showutxos(wallet, showprivkey):
+def wallet_showutxos(wallet_service, showprivkey):
     unsp = {}
     max_tries = jm_single().config.getint("POLICY", "taker_utxo_retries")
-    utxos = wallet.get_utxos_by_mixdepth(includeconfs=True)
+    utxos = wallet_service.get_utxos_by_mixdepth(include_disabled=True,
+        includeconfs=True)
     for md in utxos:
+        (enabled, disabled) = get_utxos_enabled_disabled(wallet_service, md)
+        utxo_d = []
+        if disabled != {}:
```

Comment:
This line isn't needed; if `disabled` is an empty dict, the `for` loop will just do nothing.